adopt-a-dog/blog.html

```diff
@@ -0,0 +1,85 @@
+          hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+          faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+          Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con
+          et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+          qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam
+          voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit
+          ulparume nonseca estorer spernam.
+        </p>
+      </article>
+      <article class="blog">
+        <img src="images/matt-nelson-259365-unsplash.jpg" alt="human sits with a dog" width=300>
+        <h3>How To Walk Multiple Dogs</h3>
+        <p>
+          Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+          quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium
+          hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+          faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+          Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con
+          et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+          qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam
+          voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit
+          ulparume nonseca estorer spernam.
+        </p>
+      </article>
+      <article class="blog">
+        <img src="images/wyatt-ryan-367017-unsplash.jpg" alt="human sits with a dog" width=300>
+        <h3>Teach Your Dog To Fetch!</h3>
+        <p>
+          Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+          quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium
+          hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+          faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+          Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con
+          et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+          qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam
+          voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit
+          ulparume nonseca estorer spernam.
+        </p>
+      </article>
+    </main>
+    <footer>
+      <img src="adoptadog-logo.svg" alt="site-logo" width=300>
+      <p>Aborpos nisto inci tet aut que volupti beruptiberro to ex
+esseque invenienihil imaximu scideru ptistiusam consequunt</p>
```

Comment:
Whoops...watch that formatting. It might just be a weird github thing, but make sure this is indented in.